### PR TITLE
MINOR: Fix flaky test shouldQueryOnlyActivePartitionStoresByDefault

### PR DESCRIPTION
**Pull request by cadonna**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by cadonna** [comment]
Call for review: @vvcephei @ableegoldman @guozhangwang

File: streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java [hunk 1]
```diff
@@ -136,13 +136,13 @@ public void shouldQueryOnlyActivePartitionStoresByDefault() throws Exception {
 
             final boolean kafkaStreams1IsActive = (keyQueryMetadata.activeHost().port() % 2) == 1;
 
-            // Assert that only active is able to query for a key by default
-            assertThat(kafkaStreams1IsActive ? store1.get(key) : store2.get(key), is(notNullValue()));
```
Comment (cadonna) [inline streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java:140]: Last time we forgot to move this line into the `try-catch-clause`.

File: streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java [hunk 2]
```diff
@@ -136,13 +136,13 @@ public void shouldQueryOnlyActivePartitionStoresByDefault() throws Exception {
...
             try {
                 if (kafkaStreams1IsActive) {
+                    assertThat(store1.get(key), is(notNullValue()));
                     assertThat(store2.get(key), is(nullValue()));
                 } else {
                     assertThat(store1.get(key), is(nullValue()));
+                    assertThat(store2.get(key), is(notNullValue()));
                 }
```
Comment (cadonna) [inline streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java:146]: I thought this is easier readable.

**Review by guozhangwang — APPROVED** [review]

**Comment by guozhangwang** [comment]
Cherry-picked to 2.7

**Pull request merged**